fastlane/Fastfile

```diff
@@ -111,23 +111,31 @@ platform :ios do
         scheme: "NewExpensify"
     )
 
-    upload_to_testflight(
-        api_key_path: "./ios/ios-fastlane-json-key.json",
-        distribute_external: true,
-        notify_external_testers: true,
-        changelog: "Thank you for beta testing New Expensify, this version includes bug fixes and improvements.",
-        groups: ["Beta"],
-        demo_account_required: true,
-        beta_app_review_info: {
-            contact_email: ENV["APPLE_CONTACT_EMAIL"],
-            contact_first_name: "Andrew",
-            contact_last_name: "Gable",
-            contact_phone: ENV["APPLE_CONTACT_PHONE"],
-            demo_account_name: ENV["APPLE_DEMO_EMAIL"],
-            demo_account_password: ENV["APPLE_DEMO_PASSWORD"],
-            notes: "Use the account provided. Thank you for the review."
-        }
-    )
+    begin
+      upload_to_testflight(
+          api_key_path: "./ios/ios-fastlane-json-key.json",
+          distribute_external: true,
+          notify_external_testers: true,
+          changelog: "Thank you for beta testing New Expensify, this version includes bug fixes and improvements.",
+          groups: ["Beta"],
+          demo_account_required: true,
+          beta_app_review_info: {
+              contact_email: ENV["APPLE_CONTACT_EMAIL"],
+              contact_first_name: "Andrew",
+              contact_last_name: "Gable",
+              contact_phone: ENV["APPLE_CONTACT_PHONE"],
+              demo_account_name: ENV["APPLE_DEMO_EMAIL"],
+              demo_account_password: ENV["APPLE_DEMO_PASSWORD"],
+              notes: "Use the account provided. Thank you for the review."
+          }
+      )
+    rescue Exception => e
+      if e.message.include? "Another build is in review"
+        UI.important("Another build is already in external beta review. Skipping external beta review submission")
+      else
+        raise
```

Comment:
nab - Can we do something like this instead to remove the if/else?
```
  rescue Exception => e
    raise e unless e.message.include? "Another build is in review"
    UI.important("Another build is already in external beta review. Skipping external beta review submission")
  end
```

Reply:
Yes, I think so- Is that valid ruby?

Reply:
I think that'll work, but I'm fine leaving it simple w/ the if/else since its more straightforward